docs/reference/release-notes/7.4.asciidoc

```diff
@@ -86,9 +90,9 @@ Store::
 * Allow truncation of clean translog {pull}47866[#47866]
 
 Transform::
-* [7.5][Transform] prevent assignment if any node is older than 7.4 {pull}48055[#48055] (issue: {issue}48019[#48019])
-* [Transform] prevent assignment to nodes older than 7.4 {pull}48044[#48044] (issue: {issue}48019[#48019])
-* [ML][Transforms] fix bwc serialization with 7.3 {pull}48021[#48021]
+* Prevent assignment if any node is older than 7.4 {pull}48055[#48055] (issue: {issue}48019[#48019])
+* Prevent assignment to nodes older than 7.4 {pull}48044[#48044] (issue: {issue}48019[#48019])
```

Comment:
I don't think we should document this one because it says "superseded by #48055" (which is already documented in the line above).